index.js

```diff
@@ -0,0 +1,79 @@
+console.log("Hello world");
+//task one
+let yearOfBirth = 1989;
+let yearFuture=2027;
+let age = yearFuture - yearOfBirth;
+console.log(`You will be  ${age}  years old in  ${yearFuture}  .`);
+//task two
+let dogYearOfBirth = 2015;  
+let dogYearFuture = 2027;   
+let dogYear = dogYearFuture - dogYearOfBirth; // Calculate the age in human years
```

Comment:
Very well done by adding comments here and further down. Comments will help you understand the code when you come back to it in the future, it will also help your peer developers read it faster. It's a very good practice.